M2/Macaulay2/packages/CharacteristicClasses.m2

```diff
@@ -9,7 +9,7 @@ newPackage(
 		  HomePage => "https://math.berkeley.edu/~mhelmer/"},
 	      {Name => "Christine Jost", 
 		  Email => "christine.e.jost@gmail.com"}},
-    	Headline => "CSM classes, Segre classes and the Euler characteristic for some subschemes of smooth complete toric varieties",
+	Headline => "CSM and Segre classes and the Euler characteristic for subschemes of smooth complete toric varieties",
```

Comment:
I'm trying to make all package headlines uniform, which includes removing redundant clauses. @Martin-Helmer is this change (and similar ones in SegreClasses and ToricInvariants) okay with you?

Reply:
@Martin-Helmer just a reminder about this. Are you okay with the changes to the headlines of your packages to keep then under 100 characters? We can potentially adjust it if you think 100 isn't enough.

Reply:
Yes, this is fine! Thanks Mahrud!